main/libxdmcp/APKBUILD

```diff
@@ -1,23 +1,18 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libxdmcp
-pkgver=1.1.2
-pkgrel=5
+pkgver=1.1.3
+pkgrel=0
 pkgdesc="X11 Display Manager Control Protocol library"
 url="http://xorg.freedesktop.org/"
 arch="all"
 license="MIT"
-depends=
 makedepends="xorgproto libbsd-dev util-macros xmlto"
 subpackages="$pkgname-dev $pkgname-doc"
 source="https://www.x.org/releases/individual/lib/libXdmcp-$pkgver.tar.bz2"
 
 builddir="$srcdir/libXdmcp-$pkgver"
-# secfixes:
-#   1.1.2-r3:
-#   - CVE-2017-2625
```

Comment:
Please do not remove secfixes comments. (see https://git.alpinelinux.org/alpine-secdb)